build.sbt

```diff
@@ -6,11 +6,20 @@ organization := "org.scorexfoundation"
 
 name := "sigma-state"
 
+lazy val scala212 = "2.12.10"
+lazy val scala211 = "2.11.12"
+crossScalaVersions := Seq(scala212, scala211)
+scalaVersion := scala212
+
+javacOptions ++=
```

Comment:
@aslesarenko You can set different java source and targets for different scala versions. Something along these lines should do the job:
```
javacOptions in(Compile, compile) ++= (if (scalaBinaryVersion.value == "2.11") Seq("-target", "1.7", "-source", "1.7") else Seq("-target", "8", "-source", "8" ))
scalacOptions in(Compile, compile) ++= (if (scalaBinaryVersion.value == "2.11") Seq("-release", "7") else Seq("-release", "8"))
```